### DISCUSSION TITLE
Show Category Name in the Tag list view

### DISCUSSION THREAD

**Discussion by nasirkhan** [joomla/joomla-cms#47106] [2. Cold Features]
In the category list type menu item, I can only show the list of Article names tagged with a specific article. I want to show the Category Names along with the article name. 

I found that there is category Id as `core_catid` and I can write a query to get the name of the related category. But I would like to know the recommended way to get the category name. For me, the number of queries on the tags page is already high, so what would be the optimized way to get that data? 

### Additional context

Related to  #39177

**Comment by chmst** [comment]
In general we use inner joins. If you have an article, you also have the catid and can get the categoryname with a join. 
 I am not in the code at the moment - take this carefully, it might be wrong
